setup.cfg

```diff
@@ -62,5 +62,3 @@ test =
     pytest
 
 [options.packages.find]
-where = stac_ipyleaflet
```

Comment:
FYI: The issue is that, this line was telling the python packaging system to use the files withing the `stac_ipyleaflet` directory while we want the whole directory (by default because the package name is `stac_ipyleaflet`, we will ship the directory with the name `stac_ipyleaflet`.